webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -70,6 +70,17 @@
         <ds-chip v-if="errors && errors.language" size="base" color="danger">
           <base-icon name="warning" />
         </ds-chip>
+        <ds-select
+          model="visibility"
+          icon="eye"
+          class="select-field"
+          :options="visibilityOptions"
+          :placeholder="$t('contribution.visibilitySelectText')"
+          :label="$t('contribution.visibilitySelectLabel')"
+        />
+        <ds-chip v-if="errors && errors.visibility" size="base" color="danger">
+          <base-icon name="warning" />
+        </ds-chip>
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-03-20T11:28:40Z" title="Friday, March 20th 2020, 12:28:40 pm +01:00">Mar 20, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -70,6 +70,17 @@
         <ds-chip v-if="errors && errors.language" size="base" color="danger">
           <base-icon name="warning" />
         </ds-chip>
+        <ds-select
+          model="visibility"
+          icon="eye"
+          class="select-field"
+          :options="visibilityOptions"
+          :placeholder="$t('contribution.visibilitySelectText')"
+          :label="$t('contribution.visibilitySelectLabel')"
+        />
+        <ds-chip v-if="errors && errors.visibility" size="base" color="danger">
+          <base-icon name="warning" />
+        </ds-chip>
```


aren't we trying to avoid adding new syleguide components?